pkg/executor/guest.go

```diff
@@ -30,6 +31,8 @@ import (
 	"k8s.io/client-go/util/exec"
 )
 
+const DEFAULT_CMD_TIMEOUT_IN_SECONDS = 60
```

Comment:
I think our main use of timeouts will be for enforcing "don't run forever", so the default should be quite large. (Especially if we're not yet providing a way to change the default.) Like, 15 minutes or something else hilariously big.